glide.yaml

```diff
@@ -40,3 +40,5 @@ import:
   version: v0.0.3
 - package: gopkg.in/ldap.v2
   version: v3.0.3
+- package: github.com/NYTimes/gziphandler
+  version: ~1.1.1
```

Comment:
we are putting strict versions in here. The ~ means this could install 1.1.1 or any other z-stream. We want a fixed version everywhere so we can be assured all builds are getting the same thing

Reply:
Look on line 37 - its the same except for the lowercase "ny":

- package: github.com/nytimes/gziphandler

and its at version 1.1.1.

I suspect you do not need this new NYTimes one. Look at the lock file - it even has the same hash for both  - lines 76-79:

```
- name: github.com/NYTimes/gziphandler
  version: dd0439581c7657cb652dfe5c71d7d48baf39541d
- name: github.com/nytimes/gziphandler
  version: dd0439581c7657cb652dfe5c71d7d48baf39541d
```